lib/config.js

```diff
@@ -64,30 +64,41 @@ function loadConfig(filePath) {
 }
 
 /**
+ * Loads plugin from node_modules or cache
+ * @param {string} pluginNameWithoutPrefix Name of the plugin without prefix
+ * @returns {Object} Plugin definition
+ */
+function loadPlugin(pluginNameWithoutPrefix) {
```

Comment:
Um... That's a good point. I didn't think much about command line config. There's a section in `getConfig` that does take care of providing a configuration file from command line, but I don't think there's anything that would work with environments directly from command line. I'll verify.